src/services/apiService.ts

```diff
@@ -0,0 +1,51 @@
+import { checkError, handleError } from "../utils/errorHelper";
+import localStoreService from "./localStoreService";
+
+const apiRoute = "https://reqres.in/api";
```

Comment:
I think it is better to hold those things in `.env` files

Reply:
moved